File: ControleFutebolWeb/Models/Api/TimeDtos.cs
namespace ControleFutebolWeb.Models.Api
{
    public class TimeResumoDto
    {
        public int Id { get; set; }
        public string Nome { get; set; } = string.Empty;
        public string? Cidade { get; set; }
        public string? EscudoUrl { get; set; }
        public bool EhSelecao { get; set; }
    }

    public class TimeDetalheDto : TimeResumoDto
    {
        public string? BackgroundUrl { get; set; }
        public string? CorPrincipal { get; set; }
        public string? CorSecundaria { get; set; }
        public string? CamisaUrl { get; set; }
        public string? CamisaVisitanteUrl { get; set; }
        public string? LinkTransfermarket { get; set; }
    }
}

File: AndroidApp/app/src/main/java/com/comentarista/futebol/data/remote/dto/TimeDtos.kt
package com.comentarista.futebol.data.remote.dto

import kotlinx.serialization.Serializable

@Serializable
data class TimeResumoDto(
    val id: Int,
    val nome: String,
    val cidade: String? = null,
    val escudoUrl: String? = null,
    val ehSelecao: Boolean
)

@Serializable
data class TimeDetalheDto(
    val id: Int,
    val nome: String,
    val cidade: String? = null,
    val escudoUrl: String? = null,
    val ehSelecao: Boolean,
    val backgroundUrl: String? = null,
    val corPrincipal: String? = null,
    val corSecundaria: String? = null,
    val camisaUrl: String? = null,
    val camisaVisitanteUrl: String? = null,
    val linkTransfermarket: String? = null
)
